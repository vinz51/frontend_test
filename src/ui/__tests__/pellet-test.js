import Pellet from "../pellet";
import renderer from "react-test-renderer";
import React from "react";

describe("Pellet", () => {
  describe("Snapshots", () => {
    it("renders", () => {
      const myTreeComponent = renderer.create(<Pellet />).toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });
  });
});
