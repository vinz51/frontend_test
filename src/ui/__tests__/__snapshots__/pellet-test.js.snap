// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pellet Snapshots renders 1`] = `
<div
  className="sc-AxjAm iGHMJJ"
/>
`;
